jekyll/_data/sidenav.yml

```diff
@@ -0,0 +1,217 @@
+        - name: Sample config.yml Files
+          link: /2.0/sample-config/
+        - name: Tutorials
+          link: /2.0/tutorials/
+        - name: Database Config Examples
+          link: /2.0/postgres-config/
+        - name: Example Public Repos
+          link: /2.0/example-configs/
+        - name: Testing iOS Applications
+          link: /2.0/testing-ios/
+    - name: EXECUTORS & IMAGES
+      link: /2.0/executor-intro/
+      children: 
+        - name: Choosing an Executor Type
+          link: /2.0/executor-types/
+        - name: CircleCI Docker Images
+          link: /2.0/circleci-images/
+        - name: Using Custom Images
+          link: /2.0/custom-images/
+        - name: Using Private Images
+          link: /2.0/private-images/
+    - name: ADVANCED CONFIG
+      link: /2.0/adv-config/
+      children: 
+        - name: Using Shell Scripts
+          link: /2.0/using-shell-scripts/
+        - name: Setting up Browser Testing
+          link: /2.0/browser-testing/
+        - name: Configuring Databases
+          link: /2.0/databases/
+        - name: Running Docker Commands
+          link: /2.0/building-docker-images/
+        - name: Using Docker Compose
+          link: /2.0/docker-compose/
+
+- name: Projects
+  icon: docs/project.svg
+  children: 
+    - name: settings
+      link: /2.0/settings/
+      children: 
+        - name: GitHub and Bitbucket
+          link: /2.0/gh-bb-integration/
+        - name: Building Open Source Projects
+          link: /2.0/oss/
+        - name: Using Notifications
+          link: /2.0/notifications/
+        - name: Managing API tokens
+          link: /2.0/managing-api-tokens/
+        - name: Using Environment Variables
+          link: /2.0/env-vars/
+        - name: Using Contexts
+          link: /2.0/contexts/
+        - name: Setting Up iOS Code Signing
+          link: /2.0/ios-codesigning/
+    - name: Optimizations
+      link: /2.0/optimizations/
+      children: 
+        - name: Caching Dependencies
+          link: /2.0/caching/
+        - name: Running Tests in Parallel
+          link: /2.0/parallelism-faster-jobs/
+        - name: Using Docker Layer Caching
+          link: /2.0/docker-layer-caching/
+  
+- name: Jobs
+  icon: docs/builds.svg
+  children: 
+    - name: Status
+      link: /2.0/status/
+      children:
+        - name: Using Containers
+          link: /2.0/containers/
+        - name: Adding Status Badges
+          link: /2.0/status-badges/
+        - name: Storing Build Artifacts
+          link: /2.0/artifacts/
+        - name: Debugging with SSH
+          link: /2.0/ssh-access-jobs/
+        - name: Collecting Test Metadata
+          link: /2.0/collect-test-data/
+    - name: Triggers
+      link: /2.0/triggers/
+      children: 
+        - name: Skip and Cancel Builds
+          link: /2.0/skip-build
+        - name: Using the API to Trigger Jobs
+          link: /2.0/api-job-trigger
+        - name: Orchestrating Workflows
```

Comment:
Can we change Orchestrating Workflows  to 'Using Workflows to Schedule Jobs' please (per the usability study feedback)